content/blog/2019-11-13-gep.md/index.md

```diff
@@ -0,0 +1,458 @@
++++
+title = "Making LLVM Address Calculation Safe(r)"
+extra.author = "Drew Zagieboylo"
+extra.bio = """
+  [Drew Zagieboylo](https://www.cs.cornell.edu/~dzag/) is a 3rd year PhD student researching Security, Hardware Design, and Programming Languages. He enjoys rock climbing and gaming in his free time.
+"""
++++
+
+# Memory Safety in LLVM
+
+LLVM IR code is not generally memory safe.
+While certain *obviously bad* behaviors are
+disallowed, it is not hard to write code that
+may execute out-of-bounds memory accesses at runtime.
```

Comment:
"At run time." https://capra.cs.cornell.edu/styleguide/#runtime (This also appears elsewhere in the post.)